src/app/modules/group/components/user-group-invitations/user-group-invitations.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { MessageService } from 'primeng/api';
+import { merge, of, Subject } from 'rxjs';
+import { switchMap, map } from 'rxjs/operators';
+import { GridColumn } from 'src/app/modules/shared-components/components/grid/grid.component';
+import {
+  displayResponseToast,
+  processRequestError
+} from 'src/app/modules/group/components/pending-request/pending-request-response-handling';
+import { fetchingState, readyState, isReady } from 'src/app/shared/helpers/state';
+import { GetRequestsService, PendingRequest } from '../../http-services/get-requests.service';
+import { Action, parseResults, RequestActionsService } from '../../http-services/request-actions.service';
+
+@Component({
+  selector: 'alg-user-group-invitations',
+  templateUrl: './user-group-invitations.component.html',
+  styleUrls: [ './user-group-invitations.component.scss' ]
+})
+export class UserGroupInvitationsComponent implements OnDestroy, OnInit {
+  requests: PendingRequest[] = [];
+
+  readonly columns: GridColumn[] = [
+    { field: 'group.name', header: 'TITLE' },
+    { field: 'group.type', header: 'TYPE' },
+    { field: 'at', header: 'REQUESTED ON' },
+  ];
+
+  state: 'fetching' | 'processing' | 'ready' | 'fetchingError' = 'fetching';
```

Comment:
You never put the state to 'fetching' ?🤔
And as I initially suggested, if you do not use the different between the fetching and the processing state, use only one for both.

Reply:
The 'fecthing' state is set in the subscribe of the `dataFetching` Subject via the object returned by the `fetchingState()` function

``` typescript
this.dataFetching.pipe(
      switchMap(params =>
        merge(
          of(fetchingState()), // returns { tag: 'fetching' }
          this.getRequestsService.getGroupInvitations(params.sort)
            .pipe(map(readyState))
        )
      )
    ).subscribe(
      state => {
        this.state = state.tag; // set this.state to 'fetching'
        ...
```

Reply:
Ok, sorry, I searched for the 'fetching' word. 
And for the second remark?  I see it used on line 12 of pending-join-request html but actually I don't think this disable state is still useful now we do the switchmap.

Reply:
fixed